### PR TITLE
MetadataStore method to upload multiple TUF meta files in one operation

### PR DESCRIPTION
**Pull request by endophage**

### CODE REVIEW

**Comment by NathanMcCauley** [comment]
Needs tests

File: store/httpstore.go [hunk 1]
```diff
@@ -92,8 +93,38 @@ func (s HTTPStore) SetMeta(name string, blob json.RawMessage) error {
 	return err
 }
 
+func (s HTTPStore) SetMultiMeta(metas map[string]json.RawMessage) error {
```
Comment (NathanMcCauley) [inline store/httpstore.go:96]: Is there some better type that could be used here? There must be something better that we could do than string->raw json for pushing updates.

Comment (endophage) [inline store/httpstore.go:96]: I specifically designed the stores to be dumb. They know about storing and retrieving content. Eventually I'll be switching out the `json.RawMessage` use to simply `[]byte`.

I thought for a long time about using Reader/Writer interfaces, but we always end up serializing json in memory so there isn't any gain to be had from those interfaces.

**Comment by endophage** [comment]
I've merged this code into my other PR (#5). It makes that PR larger but the two pieces have complementary moving parts and needed to be merged.

**Pull request closed without merging**